hathor/conf/settings.py

```diff
@@ -247,3 +247,6 @@ def MAXIMUM_NUMBER_OF_HALVINGS(self) -> int:
 
     # Amount in which tx min weight reaches the middle point between the minimum and maximum weight
     MIN_TX_WEIGHT_K: int = 100
+
+    # Key storage attribute to save if the full node is running a full verification
+    RUNNING_FULL_VERIFICATION_ATTRIBUTE: str = 'running_full_verification'
```

Comment:
Doesn't seem like a setting to me. Why not place it in `transaction_storage.py`?

Reply:
To be honest when I first it was on `transaction_storage.py` but then I see `STORAGE_SUBFOLDERS` on settings and this not only storage specific but it's also `compact_storage.py` specific and it was on settings. Then I decided to move.

I can move it back to `transaction_storage.py`, not a problem. It was my first thought also

Reply:
I think it should be in `transaction_storage.py` and we should move `STORAGE_SUBFOLDERS` also in the future.

Reply:
I agree with @obiyankenobi .

Reply:
Done